examples/crowd-funding/src/contract.rs

```diff
@@ -106,8 +136,50 @@ impl Contract for CrowdFunding<ViewStorageContext> {
 }
 
 impl CrowdFunding<ViewStorageContext> {
-    /// Adds a pledge from a transfer.
+    fn fungible_id() -> Result<ApplicationId, Error> {
+        let parameters = system_api::current_application_parameters();
+        let id = bcs::from_bytes(&parameters).map_err(Error::InvalidParameters)?;
+        Ok(id)
+    }
+
+    /// Adds a pledge from a remote account.
```

Comment:
This sounds a bit as if we were on the campaign chain, and received a pledge from an account on a different chain, but if I understand correctly, we're on the pledger chain here?

```suggestion
    /// Adds a pledge from a local account to the remote campaign chain.
```